drivers/include/apds99xx.h

```diff
@@ -0,0 +1,684 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_apds99xx APDS99XX proximity and ambient light sensors
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Device driver for Broadcom APDS99XX proximity and ambient light sensors
+ *
+ * The driver can be used with following Broadcom sensors:
+ * APDS9900, APDS9901, APDS9930, APDS9950, APDS9960
+ *
+ * The driver is divided into two parts:
+ *
+ * - **Basic driver** ```apds99xx```
+ *
+ *   The base driver ```apds99xx``` only supports a basic set of functions and
+ *   has therefore a small size. The procedure for retrieving new data is
+ *   polling. Ambient light and proximity sensing are supported.
+ *
+ * - **Fully functional driver** ```apds99xx_full```
+ *
+ *   The fully functional driver ```apds99xx_full``` supports all the features
+ *   supported by the base driver, as well as all other sensor features,
+ *   including interrupts and their configuration. Data-ready interrupts can be
+ *   used to retrieve data. In addition, threshold interrupts can be used and
+ *   configured.
+ *
+ * @note In addition to specifying whether the base driver ```apds99xx``` or
+ * the fully featured driver ```apds99xx_full``` should be used, the
+ * application has to declare used sensor by means of various
+ * pseudomodules as follows:
+ *
+ *      ADPS9900:     USEMODULE=apds9900
+ *      ADPS9901:     USEMODULE=apds9901
+ *      ADPS9930:     USEMODULE=apds9930
+ *      ADPS9950:     USEMODULE=apds9950
+ *      ADPS9960:     USEMODULE=apds9900
```

Comment:
This line should read `ADPS9960:     USEMODULE=apds9960`, I suppose?

Reply:
Yes. Thanks for the hint.